tests/simulator/refinement/harris.py

```
@@ -0,0 +1,223 @@
+    datahier = hierarchy_from(h5_filename=path + "/EM_E.h5", times=time, hier=datahier)
+    datahier = hierarchy_from(h5_filename=path + "/EM_B.h5", times=time, hier=datahier)
+    return datahier
+
+
+def get_hier(path):
+    return get_time(path)
+
+
+def post_advance(new_time):
+    datahier = check_hier(get_hier("diags_master"))
+    errors = test.base_test_overlaped_fields_are_equal(datahier, new_time)
```

Ruff

Local variable `errors` is assigned to but never used

Remove assignment to unused variable `errors`

(F841)